runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -119,6 +121,33 @@ export default class ExcelActionHandler extends StatefulActionHandler {
         return rowCounter;
     }
 
+    async insertColumnsBefore(input: ExcelInsertColumnsInput): Promise<void> {
+        const startingSheet = this.session.ActiveSheet.Name;
+        const workingWorksheet = this.session.Worksheets(
+            input?.worksheet ?? startingSheet
+        );
```

Comment:
merge this to const targetWorksheet = this.session.Worksheets(input?.worksheet ?? this.session.ActiveSheet.Name;)